share/golosd/config/config.ini

```diff
@@ -1,3 +1,5 @@
+mongodb-uri = mongodb://172.17.0.1:27017/Golos
+
```

Comment:
Move this line to a line after `plugin =`.
Make it optional (comment).
And add comments about what it means.